src/main/scala/com/amazon/deequ/analyzers/jdbc/ComplianceWithJdbc.scala

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. A copy of the License
+ * is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.analyzers.jdbc
+
+import JdbcUtils._
+
+object ComplianceWithJdbc extends App {
+
+  withJdbc { connection =>
+
+    val table = Table("food_des", connection)
+
+    val complianceWithFatFactorRange = JdbcCompliance("constraint name","TRUE THEN 1 ELSE 0 END) AS blakeks FROM food_des; DROP TABLE food_des; --").calculate(table)
```

Comment:
Please change this so the next person who executes this does not have any problems ;)